test/crypto.spec.ts

```diff
@@ -43,22 +96,144 @@ describe("crypto.ts", function () {
     });
   });
 
-  describe("certToPEM", function () {
-    it("should generate valid certificate", function () {
-      const cert = "-----BEGIN CERTIFICATE-----" + TEST_CERT + "-----END CERTIFICATE-----";
-      const certificate = certToPEM(cert.toString());
-      const certificateBegin = certificate.match(/BEGIN/g);
-      const certificateEnd = certificate.match(/END/g);
-      assertRequired(certificateBegin, "certificate does not have a BEGIN block");
-      assertRequired(certificateEnd, "certificate does not have an END block");
-
-      if (!(certificateBegin.length == 1 && certificateEnd.length == 1)) {
-        throw Error("Certificate should have only 1 BEGIN and 1 END block");
-      }
+  describe("keyInfoToPem", function () {
+    describe("invalid values", function () {
+      it("should throw with null", function () {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        assert.throws(() => keyInfoToPem(null as any));
+      });
+
+      it("should throw with false", function () {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Instead of disabling, let's use `@ts-expect-error`.

Reply:
We don't want to disable checking, we want to inform the linter that an error should be here.